muskathlon/models/event_compassion.py

```diff
@@ -125,6 +127,9 @@ def _compute_amount_raised(self):
             registration.amount_raised_percents = int(
                 amount_raised * 100 / registration.amount_objective)
 
+    def _compute_host(self):
+        config.get('wordpress_host')
```

Comment:
the compute method should assign the value to the record, otherwise it won't work:
```
host = config.get('wordpress_host')
if not host:
    raise error
for registration in self:
    registration.host = host
```